modules/docs/sc_param.md

```diff
@@ -36,76 +36,76 @@ The sc_param module provides methods to help you handle parameters for your stre
+| skip_anon_events                        | number | 1                                                                             | filter out events if their name can't be found in the broker cache (use 0 to accept them)                                                                                                                                                                                                                                                     | host_status(neb), service_status(neb), ba_status(bam), acknowledgement(neb)                                                                                                                                                                                                              |                                                                                                                                                                                                                                                                                                  |
+| skip_nil_id                             | number | 1                                                                             | filter out events if their ID is nil (use 0 to accept them. YOU SHOULDN'T DO THAT)                                                                                                                                                                                                                                                            | host_status(neb), service_status(neb), ba_status(bam), acknowledgement(neb)                                                                                                                                                                                                              |                                                                                                                                                                                                                                                                                                  |
```

Comment:
pourquoi on a l'option si you shouldn't do that?

Reply:
ça peut être utile quand tu sais pourquoi tu veux le faire